src/Config/CliBuilder.ts

```diff
@@ -1,8 +1,13 @@
 import { CliConfig, ConfigReader } from './ConfigReader';
 import { ClusterArgs, Helpers, logger, LogLevel } from '../Utils';
+import { SystemConfigReader } from './SystemConfigReader';
 import { CliInfo } from './CliInfo';
 import { Arguments, Argv, CommandModule } from 'yargs';
-import { runWithAsyncContext, setContext } from '@sentry/node';
+import {
+  init as initSentry,
+  runWithAsyncContext as runSentryWithAsyncContext,
+  setContext as setSentryContext
```

Comment:
Avoid using import aliases when there is no overlap.

Reply:
In my opinion, using `init`, `runWithAsyncContext`, `setContext` decreases code readability. These function names don't say a lot about what they do – they are just too generic.

For example, using `Sentry.init` would explain more about method usage. But wildcard imports are forbidden by our eslint rules.

So I reexported Sentry methods from utils in order to overcome eslint restriction. Does it work for you?

Reply:
Actually, I don't see any issues. You are aware of where they come from. They are not included globally like in C++ or Python.

Anyway, the use of wildcard import is prohibited, but importing a module as a namespace is allowed.

```ts
import sentry from '@sentry/node';

sentry.init({
  // ..
})
```

Reply:
> Actually, I don't see any issues. You are aware of where they come from. They are not included globally like in C++ or Python.

Yep, but still requires navigation to the top and checking source

Reply:
> Yep, but still requires navigation to the top and checking source

The same applicable to aliasing since the dev is not aware of such functions and where they came from.

Reply:
I think situations are different:

1. Function `initSentry` clearly says that it initialising Sentry instance. The rest is a matter of trust.
2. Function `init` doesn't describe it's behaviour. So it's nearly impossible not to check where it came from in order to understand its behaviour.